### PR TITLE
Add cloudzone to pro_services

### PR DESCRIPTION
**Pull request by ohadse-cldze**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by ohadse-cldze** [comment]
@googlebot I fixed it.

File: data/companies.yml [hunk 1]
```diff
@@ -221,10 +221,13 @@ integrations:
...
-    url: "https://istio.io/"
-    description: "Submit a PR to add your company here!"
+  - name: "CloudZone"
+    logo: "/logos/cloudzone.svg"
+    url: "https://cloudzone.io/"
+    description:
+      - "At CloudZone, we’ve been actively involved with Kubernetes and Istio since the very first editions came out." 
+      - "In today’s fast-paced, ever-changing world, we continue to research the field keeping up to date so that we can bring you the very latest thinking and solutions, whether we are supporting you through the provision of consulting, design, training or production support services."
+      - "Our certified Kubernetes administrators and cloud experts are highly skilled in building production-grade Kubernetes systems with the latest Cloud native tools for automating application deployment, scaling and management, together with the open source Istio service mesh for connecting, securing, controlling and observing services."
   - name: "Your Name Here"
```
Comment (craigbox) [inline data/companies.yml:231]: Please add a single line "description" and change the heading to "details".  

(See https://github.com/istio/istio.io/pull/10024/files for the fixed template, with my apologies)

**Review by craigbox — CHANGES_REQUESTED** [review]

**Comment by ohadse-cldze** [comment]
@googlebot I fixed it.

**Comment by ohadse-cldze** [comment]
@googlebot I fixed it.

File: data/companies.yml (hunk 1 above)
Comment (ohadse-cldze) [reply data/companies.yml]: I've added "details"

**Comment by craigbox** [comment]
PTAL at https://deploy-preview-10022--preliminary-istio.netlify.app/latest/about/ecosystem. The `description` and `details` headers need to be flipped around.

Also need CLA signed by `50926100+ohad-work@users.noreply.github.com` (or [edit the commits](https://gist.github.com/trey/9588090) to use your other e-mail address)

**Review by craigbox — APPROVED** [review]

**Comment by craigbox** [comment]
add "ohad-work" to your list of usernames in the CLA?

**Comment by ohadse-cldze** [comment]
@googlebot I fixed it.

**Comment by craigbox** [comment]
🎉 thanks!

**Pull request merged**